go-packages/meep-http-logger/httpLogger.go

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2020  InterDigital Communications, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package httpLogger
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-logger"
+	ms "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-metric-store"
+)
+
+var nextUniqueId int32 = 1
+var redisDBAddr string = "meep-redis-master:6379"
+var influxDBAddr string = "http://meep-influxdb:8086"
+var metricStore *ms.MetricStore
+var logComponent = ""
+
+const DirectionRX = "RX"
+const DirectionTX = "TX"
+
+func ReInit(loggerName string, currentStoreName string) error {
+
+	log.Info("Reinitialisation of http logger with: ", currentStoreName, " for ", loggerName)
+	logComponent = loggerName
+	if currentStoreName != "" {
+		//currentStoreName located in NBI of RNIS populated by SBI upon new activation
+		var err error
+		metricStore, err = ms.NewMetricStore(currentStoreName, influxDBAddr, redisDBAddr)
```

Comment:
Why create a new metric store here? You can create it once and then use SetStore() to set the store name to use.

Reply:
not addressed... metric store added when a deployment is sent, deleted when terminated. I guess either way are valid.... no change for now